cityofaustin/atd-moped
# Prevent saving empty descriptions, update button size

## Associated issues

https://github.com/cityofaustin/atd-data-tech/issues/17027

## Testing

**URL to test:** 

https://deploy-preview-1337--atd-moped-main.netlify.app/moped/projects/1933


**Steps to test:**

Try to save an empty description

I also updated the button sizes to all be medium (the default material ui button size)

And I fixed a bug Tilly found, https://github.com/cityofaustin/atd-moped/pull/1308#issuecomment-2087798693, you should be able to clear the public process field now. 

---
#### Ship list
